x-pack/plugins/synthetics/server/routes/create_route_with_auth.ts

```diff
@@ -13,18 +13,16 @@ import {
 } from '../../common/constants';
 import { SyntheticsRestApiRouteFactory, SyntheticsRoute, SyntheticsRouteHandler } from './types';
 
-function getWriteAccessFlag(method: string, writeAccessOverride?: boolean, writeAccess?: boolean) {
-  // if route includes an override, skip write-only access with `undefined`
-  // otherwise, if route is not a GET, require write access
-  // if route is get, use writeAccess value with `false` as default
-  return writeAccessOverride === true ? undefined : method !== 'GET' ? true : writeAccess ?? false;
+function getDefaultWriteAccessFlag(method: string) {
```

Comment:
I wonder if you should use https://github.com/cauemarcondes/kibana/blob/7f17340dfdb191dabc9a78c3b73d8e87959f2c0c/x-pack/plugins/synthetics/server/routes/types.ts#L33 instead of `string` on the **method** type.

Reply:
[92e538f](https://github.com/elastic/kibana/pull/177088/commits/92e538f776d17322d716b07f9a36beff0f1f5c9a)